src/components/UserInfo.js

```diff
@@ -11,11 +12,24 @@ export default class UserInfo {
     const userObject = {};
     userObject["profileName"] = this._nameElement.textContent;
     userObject["description"] = this._jobElement.textContent;
+    userObject["imgLink"] = this._imgElement.src;
```

Comment:
Please don't return the avatar here, since it's controlled by a different form. You can create a separate method `getProfileImage`